core/templates/pages/skill-editor-page/modal-templates/add-misconception-modal.controller.spec.ts

```diff
@@ -121,4 +121,26 @@ describe('Add Misconception Modal Controller', function() {
     $scope.saveMisconception();
     expect($uibModalInstance.close).not.toHaveBeenCalled();
   });
+
+  it('should check content length in misconception notes within 1000 chars',
+    function() {
+      var contentText = 'This is just a sample text.';
+
+      $scope.misconceptionNotes = contentText;
+      expect($scope.isMisconceptionNotesHtmlLengthExceeded()).toBe(false);
+
+      $scope.misconceptionNotes = contentText.repeat(50);
+      expect($scope.isMisconceptionNotesHtmlLengthExceeded()).toBe(true);
```

Comment:
Might be more clear to do something like this (for all of the tests)!
```suggestion
      $scope.misconceptionNotes = 'a'.repeat(1000);
      expect($scope.isMisconceptionNotesHtmlLengthExceeded()).toBe(false);

      $scope.misconceptionNotes = 'a'.repeat(1001);
      expect($scope.isMisconceptionNotesHtmlLengthExceeded()).toBe(true);
```

Reply:
Sure looks good.  I will apply the changes

Reply:
Thanks for the lightning fast response @ashish-patwal!!